src/ReflectionFile.php

```diff
@@ -50,6 +50,9 @@ class ReflectionFile
      */
     public function __construct($fileName, $topLevelNodes = null)
     {
+        if (!is_string($fileName)) {
```

Comment:
Ok... I searched the code, and (outside of `tests/`) nothing within ParserReflection instantiates `Go\ParserReflection\ReflectionFile`, and ever use of it is annotated as `string`: The constructor is annotated `@param string`, the class property is annotated `@var string`, and the accessor is annotated `@return string`. I can only assume that this a plea for enforcing type safety rather than warning that badly typed data is being redirected here. I happen to be a fan of enforcing types where appropriate, so I see no harm in leaving this change, and the matching one for `ReflectionFileNamespace` intact.

👎 Keeping change in PR pending @lisachenko's approval.